dds/DCPS/RTPS/RtpsDiscoveryConfig.h

```diff
@@ -418,7 +418,7 @@ class OpenDDS_Rtps_Export RtpsDiscoveryConfig : public OpenDDS::DCPS::RcObject {
   void spdp_send_addrs(const AddrVec& addrs)
   {
     ACE_GUARD(ACE_Thread_Mutex, g, lock_);
-    spdp_send_addrs_ = addrs;
+    spdp_send_addrs_.insert(spdp_send_addrs_.end(), addrs.begin(), addrs.end());
```

Comment:
This will break existing usage patterns as users are expecting set instead of insert semantics.